mmv1/third_party/terraform/tests/data_source_google_iam_policy_test.go

```diff
@@ -0,0 +1,66 @@
+package google
+
+import (
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+)
+
+func TestAccComputeImageIamPolicy_googleIamPolicyDataSource(t *testing.T) {
+	t.Parallel()
+
+	context := map[string]interface{}{
+		"random_suffix": randString(t, 10),
+		"role":          "roles/compute.imageUser",
+		"member":        "user:admin@hashicorptest.com",
```

Comment:
This seems potentially a little odd to hardcode since developers might be running these tests on arbitrary projects.... but I see that this is already hardcoded in a bunch of other tests so I guess it's fine 😂